packages/auth/src/openapi/specs/id-provider.yaml

```diff
@@ -39,6 +39,8 @@ paths:
               description: Interaction id
         '401':
           description: Unauthorized
+        '500':
+          description: Request denied
```

Comment:
Since this applies to all requests, we can probably leave it out for now. (later we can just add 500 as generic error we can specify for each response)